Makefile

```diff
@@ -28,10 +34,33 @@ $(GOLANGCI_LINT): $(TOOLS_DIR)
 		GOBIN=$(TOOLS_DIR) go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.1 ;\
 	}
 
+$(KUSTOMIZE): $(TOOLS_DIR)
+	@{ \
+		set -e ;\
+		GOBIN=$(TOOLS_DIR) go get sigs.k8s.io/kustomize/kustomize/v3 ;\
```

Comment:
We should be using `go install`, `go get` will be deprecated for compiling binaries from source [1]. Can kustomize be installed with `go install`?

[1] https://golang.org/doc/go-get-install-deprecation